lib/index.js

```diff
@@ -91,22 +92,22 @@ MaplibreGeocoder.prototype = {
     clearAndBlurOnEsc: false,
     clearOnBlur: false,
     getItemValue: function (item) {
-      return typeof item === "string" ? item : item.place_name;
+      return item.text !== undefined ? item.text : item.place_name;
```

Comment:
Is item guaranteed to be an object now? Will never be undefined?

I don't see know where this is used, so it's hard to know what's guaranteed, especially without TS.

Reply:
Another thought on this, couldn't this actually break customer if they don't get the packages aligned and item is not an object?

Reply:
will add a check for that